txclib/parsers.py

```diff
@@ -261,6 +265,12 @@ def set_main_parser():
                                    "list of available i18n types, see "
                                    "http://docs.transifex.com/formats/"
                                    ))
+    main_parser.add_argument("--auto-local", action="store_true",
+                             dest="local", default=False,
+                             help="Alias of auto-local subcommands")
```

Comment:
@SofiaMargariti is the wording of this finalized? I think I would prefer singular here (`Alias of auto-local subcommand`), unless there is a reason we are using plural.

Reply:
You are right that was a typo, although the naming is not finalized yet